docker/scripts/setenv.bat

```diff
@@ -0,0 +1,10 @@
+rem update this file to match your local environment
+
+set MVN=C:\opt\Apache\Maven\apache-maven-3.8.3\bin\mvn
```

Comment:
Eigenlijk is deze (en JAVA_HOME als die niet al gezet is) de enige die per gebruiker anders zal zijn, die in een aparte setlocalenv.bat zetten en die in de .gitignore zetten?